tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py27
+
+[testenv]
+usedevelop = true
+deps =
+    -rrequirements.txt
+    pyftpdlib
+    pytest-xdist
+commands =
+    py.test {posargs:tests}
```

Comment:
i'm not sure this matters, but this is slightly different than what travis is currently doing, which is:

```
script: make test PYTESTOPTS="--tb=short" TESTS="tests/test_*.py tests/configuration/test_*.py tests/logger/test_*.py tests/checker/test_httpbin.py"
```

`make test`, in turn, does this:

```
test:   localbuild
        env LANG=en_US.utf-8 $(PYTHON) -m pytest $(PYTESTOPTS) $(TESTOPTS) $(TESTS)
```

And those variables are:

```
TESTS ?= tests
# set test options, eg. to "--verbose"
TESTOPTS=
# Pytest options:
# - use multiple processes
# - write test results in file
PYTESTOPTS:=-n 4 --resultlog=testresults.txt
```

Arguably, this is a mess, and I prefer to have your clean tox interface... but is there something in there we're missing we'd like to add?

in any case, i think it would be preferable to *switch* to tox instead of *adding* it. that would mean changing the `.travis.yml` configuration along with the `Makefile`.

arguably, we should also make `python setup.py test` actually work (and that's possible!) but that may be out of scope here (as it doesn't currently work).